src/Ombi.Core/Engine/V2/MultiSearchEngine.cs

```diff
@@ -43,72 +44,92 @@ public async Task<List<MultiSearchResult>> MultiSearch(string searchTerm, MultiS
             var lang = await DefaultLanguageCode(null);
             var model = new List<MultiSearchResult>();
 
-            var movieDbData = (await _movieDbApi.MultiSearch(searchTerm, lang, cancellationToken)).results;
-
             var lidarrSettings = await _lidarrSettings.GetSettingsAsync();
             if (lidarrSettings.Enabled && filter.Music)
             {
-                var artistResult = await _musicApi.SearchArtist(searchTerm);
-                foreach (var artist in artistResult)
+                var lidarSearchResult = await _lidarrApi.Search(searchTerm, lidarrSettings.ApiKey, lidarrSettings.FullUri);
+                foreach (var search_result in lidarSearchResult)
                 {
-                    model.Add(new MultiSearchResult
+                    if (search_result.artist != null) 
                     {
-                        MediaType = "Artist",
-                        Title = artist.Name,
-                        Id = artist.Id
-                    });
+                        model.Add(new MultiSearchResult
+                        {
+                            MediaType = "Artist",
+                            Title = search_result.artist.artistName,
+                            Id = search_result.artist.foreignArtistId,
+                            Poster = search_result.artist.remotePoster,
+                            Monitored = search_result.artist.monitored
+                            
+                        });
+                    } else if (search_result.album != null)
+                    {
+                        model.Add(new MultiSearchResult
+                        {
+                            MediaType = "Album",
+                            Title = search_result.album.title,
+                            Id = search_result.album.foreignAlbumId,
+                            Poster = search_result.album.remoteCover,
+                            Monitored = search_result.album.monitored
+                        });
+                    }
+
                 }
             }
 
-            foreach (var multiSearch in movieDbData)
+            if (filter.Movies)
```

Comment:
This is not going to work, if we just have TV in the filter it’s not going to show anything